app/assets/stylesheets/application.scss

```diff
@@ -57,22 +49,12 @@
   justify-content: space-around;
   margin-bottom: 69px;
 }
-#card{
+.card{
```

Comment:
looks like this is overriding a bootstrap class. Perhaps add a prefix to this class so that it's a custom class and target the element with this class.

Reply:
We removed the id, and added class name